tests/transforms_tests/image_tests/test_rotate.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+import numpy as np
+
+from chainer import testing
+from chainercv.transforms import rotate
+
+
+@testing.parameterize(*testing.product({
+    'k': [0, 1, 2, 3]
+}))
+class TestRotate(unittest.TestCase):
+
+    def test_rotate(self):
+        img = np.random.uniform(size=(3, 24, 24))
```

Comment:
How about using non-square image for test?

Reply:
updated